corpus/convert.py

```diff
@@ -182,3 +183,45 @@ def run(self):
                 segment.orth = line.strip()
 
         c.dump(tk.uncached_path(self.out_corpus))
+
+
+class CorpusToTextDictJob(Job):
+    """
+    Extract the Text from a Bliss corpus to fit a "{key: text}" structure (e.g. for RETURNN)
+    """
+
+    def __init__(self, corpus, segments=None):
+        """
+        :param Path corpus: bliss corpus file
+        :param Path|None segments: a segment file as optional whitelist
```

Comment:
add maybe option for blacklist? like FilterCorpusBySegmentsJob (I think)

Reply:
But then we would need to add it to all jobs that take segments for consistency, well of course we can do that...

Reply:
I created an issue for this, then we are reminded of this and can do it later on for all jobs.

Reply:
yes fine with me